src/TemplateLibrary/index.js

```diff
@@ -1,9 +1,19 @@
 import _ from 'lodash';
+
+/* React */
 import React from 'react';
 import PropTypes from 'prop-types';
+
+/* Styling */
 import styled from 'styled-components';
 import { Button, Card, Input } from 'semantic-ui-react';
 
+/* Cicero */
+import { TemplateLibrary } from '@accordproject/cicero-core';
+import * as ciceroPackageJson from '@accordproject/cicero-core/package.json';
+const ciceroVersion = ciceroPackageJson.version;
```

Comment:
lines 13 and 14 can be replaced with `import { version as ciceroVersion } from '@accordproject/cicero-core/package.json';`